src/test/TestGhoCcipSteward.t.sol

```diff
@@ -252,6 +252,105 @@ contract TestGhoCcipSteward is TestGhoBase {
     );
   }
 
+  function testDisableRateLimitFromNonZeroToZero() public {
+    RateLimiter.TokenBucket memory outboundConfig = MockUpgradeableLockReleaseTokenPool(
+      GHO_TOKEN_POOL
+    ).getCurrentOutboundRateLimiterState(remoteChainSelector);
+    RateLimiter.TokenBucket memory inboundConfig = MockUpgradeableLockReleaseTokenPool(
+      GHO_TOKEN_POOL
+    ).getCurrentInboundRateLimiterState(remoteChainSelector);
+
+    assertTrue(outboundConfig.isEnabled);
+    assertGt(outboundConfig.capacity, 0);
+    assertGt(outboundConfig.rate, 0);
+
+    assertTrue(inboundConfig.isEnabled);
+    assertGt(inboundConfig.capacity, 0);
+    assertGt(inboundConfig.rate, 0);
+
+    RateLimiter.Config memory disableLimitConfig = RateLimiter.Config({
```

Comment:
we can reuse `outboundConfig` and `inboundConfig` no?

Reply:
not for `disableLimitConfig` since they're non zero (asserted on the lines above) & they're [not set the same](https://github.com/aave/gho-core/blob/test/disable-rate-limit/src/test/TestGhoBase.t.sol#L336-L353) in the test setup

Reply:
oh as in zero them out & use, yes